### DISCUSSION TITLE
Can't connect Xbox controller via USB sharing

### DISCUSSION THREAD

**Discussion by Glydric** [utmapp/UTM#4453] [UTM Help (macOS)] [answered]
**Describe the issue**  
I was trying to run gamepass on xbox app, so I connected xbox controller on windows 11 by cable and then obtain the error LIBUSB_ERROR_NO_DEVICE, is not random but trying more times it continues to pop

**Configuration**  
* UTM Version: 4.0.6
* macOS Version: 12.6
* Mac Chip (Intel, M1, ...): M1 pro

<img width="383" alt="image" src="https://user-images.githubusercontent.com/56474420/192387001-0c849408-ffcc-4544-a8bd-8c41fc77e8af.png">

**Comment by Glydric** [comment, marked as answer]
After further test i figured out that the controller is not really connected to macos but recognised by utm (like I know that the device is connected and charging but I'm not supposed to use it by cable), for the apple support seems that the only method to connect the controller is through bluetooth 
https://support.apple.com/en-gb/HT211232

**Comment by conath** [comment]
Side note: currently UTM does not include accelerated graphics for Windows Guests, so you might be better off with another VM host. AFAIK the app Parallels Desktop is able to run Windows games.

**Comment by Glydric** [comment]
in reality the streaming through xbox app works and display my console, but i cannot play because i cannot connect controller

**Reply by conath** [reply]
Oh I see, you are streaming the gameplay. That makes sense.

**Reply by Glydric** [reply]
I installed xbox app through exe file downloaded from microsoft and connected my account
<img width="1624" alt="Screenshot 2022-09-27 at 10 34 19 AM" src="https://user-images.githubusercontent.com/56474420/192476467-953c026d-8420-4de4-bc77-98a1beb27b8d.png">
